src/main/kotlin/app/hashers/CommitHasher.kt

```diff
@@ -213,32 +175,15 @@ class RepoHasher(private val localRepo: LocalRepo, private val api: Api,
     }
 
     fun update() {
-        if (!RepoHelper.isValidRepo(localRepo.path)) {
-            Logger.error("Invalid repo $localRepo")
-            return
-        }
-
-        println("Hashing $localRepo...")
-        localRepo.parseGitConfig(gitRepo.config)
-        repo.userEmail = localRepo.author.email
-        calculateRepoRehashes()
-
-        if (isKnownRepo()) {
-            getRepoFromServer()
-
-            // Delete missing commits. If found at least one common commit
-            // then next commits are not deleted because hash of a commit
-            // calculated including hashes of its parents.
-            val firstOverlapCommit = findFirstOverlappingCommit()
-            val deletedCommits = repo.commits
-                .takeWhile { it.rehash != firstOverlapCommit?.rehash }
-            deleteCommitsOnServer(deletedCommits)
-        }
-
+        // Delete missing local commits. If found at least one common commit
```

Comment:
This sounds confusing. You are deleting commits on server, not locally.

Reply:
Ok.